components/scripts/gradle/01-validate-incremental-building.sh

```diff
@@ -149,12 +149,10 @@ execute_build() {
 
 print_gradle_command() {
   local scan_arg
-  if [[ "${build_scan_publishing_mode}" == "on" ]]; then
-    scan_arg="--scan"
-  else
-    scan_arg="-Dscan.dump"
+  if [[ "${build_scan_publishing_mode}" != "on" ]]; then
```

Comment:
It looks like we typically check for `on` vs `off` in the codebase.

```suggestion
  if [[ "${build_scan_publishing_mode}" == "off" ]]; then
```